examples/notebooks/Upper_Air_BUFR_Soundings.ipynb

```diff
@@ -340,7 +335,7 @@
    "source": [
     "### Convert Units\n",
     "\n",
-    "We need to modify the units several of the data parameters are returned in.  Here we convert Temperature from Fahrenheit to Celcius, convert pressure to milibars, and extract wind for both the u and v directional components in Knots and Radians. "
+    "We need to modify the units several of the data parameters are returned in.  Here we convert Temperature units from Fahrenheit to Celcius, convert pressure to milibars, and extract wind for both the u and v directional components in Knots and Radians. "
```

Comment:
"Here we convert Temperature and Dewpoint units..."

Also, it looks like we're converting them from Kelvin to Celcius

Reply:
You're right. I don't know why I put F in the comment 😵‍💫